student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student
```

Comment:
- [ ] Same here as the previous reviewer mentioned, kindly make the Student class inherit from the person class. 
```
class Student < Person
```

Reply:
done